docs/source/API/core/policies/TeamHandleConcept.rst

```diff
@@ -0,0 +1,174 @@
+``TeamHandleConcept``
+=====================
+
+.. role:: cppkokkos(code)
+    :language: cppkokkos
+
+Header File: ``<Kokkos_Core.hpp>``
+
+TeamHandleConcept defines the concept for the ``member_type`` of ``TeamPolicy`` and ``TeamTask``.
+The actual type is defined through the policies, but meets the following API requirements.
+Note that the specific classes are only part of the public API as provided by ``TeamPolicy`` and
+``TeamTask`` and only their parts as defined by the ``TeamHandleConcept``.
+The actual name of the class, as well as potential template parameters, existing
+constructors, member functions going beyond the concept, etc., are NOT part of the public Kokkos API
+and are thus subject to change.
+
```

Comment:
There should be no comment on things that are not part of the public API.  Please remove non-public API material.  Perhaps you could simply delete line 13 -- it's not very informative, in any case.

Reply:
I think we should keep this because it stresses and reminds users that this is just the concept. Repetita iuvant, especially for things as delicate as these.